### ISSUE TITLE
other datatypes than string and number

### ISSUE THREAD

**Issue by sebastibe** [mvasilkov/django-google-charts#2]
I am not sure to see how to use other datatypes than "string" and "number" as you have to use the javascript objects, for example for "datetime". Anything I would have missed?

**Comment by mvasilkov** [comment]
I usually format dates as strings; I’m not sure if Google charts API even accepts JS Date objects.

**Comment by sebastibe** [comment]
Yes they accept: 'string' 'number' 'boolean' 'date' 'datetime' 'timeofday'.
My issue comes from the fact I am trying to use the Annotated Timeline (http://code.google.com/apis/chart/interactive/docs/gallery/annotatedtimeline.html) which only accepts 'date' and 'datetime' for the first column.
